### PR TITLE
AC: support kaldi asr models

### PR DESCRIPTION
**Pull request by eaidova**
TO DO:

- [x]  converter
- [x] ark-files reader
- [x] output decoder
- [x] SER metric
- [x] documentation

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/metrics/speech_recognition.py [hunk 1]
```diff
@@ -74,3 +75,30 @@ def evaluate(self, annotations, predictions):
...
+    def configure(self):
+        if isinstance(editdistance, UnsupportedPackage):
+            editdistance.raise_error(self.__provider__)
+        self.length = 0
+        self.score = 0
+        self._wer_measure = SpeechRecognitionWER({'type': 'wer'}, self.dataset)
+
+    def update(self, annotation, prediction):
+        wer = self._wer_measure.update(annotation, prediction)
+        self.score += str(wer != 0)
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/speech_recognition.py:95]:
```suggestion
        self.score += int(wer != 0)
```

File: tools/accuracy_checker/accuracy_checker/adapters/README.md [hunk 2]
```diff
@@ -394,3 +394,18 @@ AccuracyChecker supports following set of adapters:
 * `background_matting` - converts output of background matting model to `BackgroundMattingPrediction`.
 * `noise_suppression` - converts output of audio denoising model to `NoiseSuppressionPrediction`.
   * `output_blob` - name of output layer with processed signal (Optional, if not provided, first found output from model will be used).
+* `kaldi_latgen_faster_mapped` - decodes output Kaldi\* automatic speech recognition model using lattice generation approach with transition model to `CharcterRecognitionPrediction`.
+  **Important note** This adapter requires [Kaldi\* installation](https://kaldi-asr.org/doc/install.html)(we recommend to use `67db30cc` commit)
+  and providing path to directory with compiled executable apps: `latgen-faster-mapped`, `lattice-scale`, `lattice-add-penalty`, `lattice-best-path`.
+  Path directory can be provided using `--kaldi_bin_dir` commandline argument or `KALDI_BIN_DIR` environment variable.
+  * `fst_file` - Weighted Finite-State Transducers (WFST) state graph file.
+  *`words_file` - words table file.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/adapters/README.md:402]:
```suggestion
  * `words_file` - words table file.
```

File: tools/accuracy_checker/accuracy_checker/adapters/README.md [hunk 3]
```diff
@@ -394,3 +394,18 @@ AccuracyChecker supports following set of adapters:
 * `background_matting` - converts output of background matting model to `BackgroundMattingPrediction`.
 * `noise_suppression` - converts output of audio denoising model to `NoiseSuppressionPrediction`.
   * `output_blob` - name of output layer with processed signal (Optional, if not provided, first found output from model will be used).
+* `kaldi_latgen_faster_mapped` - decodes output Kaldi\* automatic speech recognition model using lattice generation approach with transition model to `CharcterRecognitionPrediction`.
+  **Important note** This adapter requires [Kaldi\* installation](https://kaldi-asr.org/doc/install.html)(we recommend to use `67db30cc` commit)
+  and providing path to directory with compiled executable apps: `latgen-faster-mapped`, `lattice-scale`, `lattice-add-penalty`, `lattice-best-path`.
+  Path directory can be provided using `--kaldi_bin_dir` commandline argument or `KALDI_BIN_DIR` environment variable.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/adapters/README.md:400]:
```suggestion
  Path directory should be provided using `--kaldi_bin_dir` commandline argument or `KALDI_BIN_DIR` environment variable.
```

**Review by Anna-Mironova — APPROVED** [review]

**Comment by eaidova** [comment]
Jenkins please retry a build

**Pull request merged**